contracts/forwarder/Forwarder.sol

```diff
@@ -70,7 +72,7 @@ contract Forwarder is IForwarder {
 
 
     function _verifyNonce(ForwardRequest memory req) internal view {
-        require(nonces[req.from] == req.nonce, "nonce mismatch");
+        require(nonces[req.from] == req.nonce, "FWD: nonce mismatch");
```

Comment:
If we are adding a prefix, I would add error numbers, what do you think?
Like `Error: FWD-401: nonce mismatch`
would make for easier documentation, searching etc. and can get away with shorter error messages

Reply:
string is unique. I did want to scope all forwarder errors with FWD, but I hate numbers - they add nothing.